server/util/truncateUtil.js

```diff
@@ -0,0 +1,6 @@
+const truncateUtil = function(num, fixed = null) {
```

Comment:
Let's rename this file `numberUtils` and export a named function called `truncateTo`. And instead of using this regepx, I think we do something simple like:
```js
const truncateTo = (num, precision = 0) => {
  const factor = 10 ** precision;
  return Math.floor(num * factor) / factor;
};
```

I think this will do the same thing and be a little faster.